libs/blocks/icon-block/icon-block.css

```diff
@@ -74,6 +73,16 @@
   width: auto;
 }
 
+.icon-block.inline .icon-area,
+.icon-block.inline .icon-area picture img {
+  min-width:80px;
+}
+
+.icon-block.inline.bio .icon-area,
+.icon-block.inline.bio .icon-area picture img {
+  min-width:60px;
```

Comment:
```suggestion
  min-width: var(--xl-icon);
```

Reply:
Technically the xl-icon is 64px but maybe we need that re-alignment on consistency anyways